lib/avatarService.ts

```diff
@@ -29,7 +29,7 @@ export class AvatarService {
   }
 
   /**
-   * Returns promise of the Avatar TruffleContract.
+   * Returns promise of the Avatar Truffle contract wrapper.
```

Comment:
Why arc.js needs the  truffle contracts ?
if there is a good reason  ..please document that.
Where is the dependency point of arc.js on truffle ?

Reply:
@orenyodfat 

> Why arc.js needs the truffle contracts ?

1)  Arc.js uses them intternally
2) They are made readily accessible to users to facilitate operations that Arc.js does wrap, either now or perhaps ever.
3) The most important dependencies on truffle might be the `ContractWrapperBaseFactory` class, the `hydrate` methods in `ContractWrapperBase` and all of the wrapper methods that call Arc contract methods.

At your suggestion, I added some docs to explain how to obtain truffle contract and web3.